### PR TITLE
VEP JSON output - remove frequencies of non-matching co-located alleles

### PR DESCRIPTION
**Pull request by helensch**
This is related to  #938. For further details and examples see: ENSVAR-4006

### CODE REVIEW

File: modules/Bio/EnsEMBL/VEP/OutputFactory/JSON.pm [hunk 1]
```diff
@@ -124,6 +124,13 @@ my @LIST_FIELDS = qw(
   var_synonyms
 );
 
+my @FREQ_FIELDS = qw(
```
Comment (at7) [inline modules/Bio/EnsEMBL/VEP/OutputFactory/JSON.pm:127]: Could you please use the new array also in this loop here https://github.com/Ensembl/ensembl-vep/blob/release/103/modules/Bio/EnsEMBL/VEP/OutputFactory/JSON.pm#L396

**Review by at7 — APPROVED** [review]
This fixes an oversight when working with multi-allelic variants where only a subset of the alleles is provided by the user. It is absolutely correct to remove all remaining frequency keys from the existing hash.

**Comment by at7** [comment]
I realised that I already approved the PR. Sorry for the confusion. I will approve it again after you had a chance to update the array.

**Comment by helensch** [comment]
@at7  thanks for the review. I have modified to use @FREQ_FIELDS when setting frequencies

**Review by at7 — APPROVED** [review]

**Comment by at7** [comment]
merged into master and release/104

**Pull request merged**